app/src/ui/check-runs/ci-check-run-rerun-dialog.tsx

```diff
@@ -123,95 +122,135 @@ export class CICheckRunRerunDialog extends React.Component<
         cr.checkSuiteId !== null &&
         rerequestableCheckSuiteIds.includes(cr.checkSuiteId)
     )
-    const nonRerunnable = this.props.checkRuns.filter(
+    const nonRerunnable = checkRunsToConsider.filter(
       cr =>
         cr.checkSuiteId === null ||
-        !rerequestableCheckSuiteIds.includes(cr.checkSuiteId) ||
-        (this.props.failedOnly && cr.conclusion === APICheckConclusion.Failure)
+        !rerequestableCheckSuiteIds.includes(cr.checkSuiteId)
     )
 
     this.setState({ loadingCheckSuites: false, rerunnable, nonRerunnable })
   }
 
   private renderRerunnableJobsList = () => {
-    if (this.state.loadingCheckSuites) {
-      return (
-        <div className="loading-rerun-checks">
-          <img src={BlankSlateImage} className="blankslate-image" />
-          <div className="title">Please wait</div>
-          <div className="call-to-action">
-            Determining which checks can be re-run.
-          </div>
-        </div>
-      )
+    if (this.state.rerunnable.length === 0) {
+      return null
     }
 
     return (
       <div className="ci-check-run-list check-run-rerun-list">
-        {this.state.rerunnable.length > 0 ? (
-          <CICheckRunList
-            checkRuns={this.state.rerunnable}
-            loadingActionLogs={false}
-            loadingActionWorkflows={false}
-            notExpandable={true}
-          />
-        ) : null}
+        <CICheckRunList
+          checkRuns={this.state.rerunnable}
+          loadingActionLogs={false}
+          loadingActionWorkflows={false}
+          notExpandable={true}
+          isCondensedView={true}
+        />
       </div>
     )
   }
 
-  private renderRerunInfo = () => {
+  private renderRerunDependentsMessage = () => {
+    if (this.state.rerunnable.length === 0) {
+      return null
+    }
+
+    const name =
+      this.props.checkRuns.length === 1 ? (
+        <strong>{this.props.checkRuns[0].name}</strong>
+      ) : (
+        'these workflows'
+      )
+    const dependentAdj = this.props.checkRuns.length === 1 ? "it's" : 'their'
```

Comment:
Should this be [its](https://www.grammarly.com/blog/its-vs-its/)?

```suggestion
    const dependentAdj = this.props.checkRuns.length === 1 ? "its" : 'their'
```

Reply:
> Should this be [its](https://www.grammarly.com/blog/its-vs-its/)?
> 
> 
> 
> ```suggestion
> 
>     const dependentAdj = this.props.checkRuns.length === 1 ? "its" : 'their'
> 
> ```